go/common/headers.go

```diff
@@ -20,18 +21,21 @@ var hasherPool = sync.Pool{
 // Header is a public / plaintext struct that holds common properties of rollups batches.
 // Making changes to this struct will require GRPC + GRPC Converters regen
 type Header struct {
-	ParentHash  L2RootHash
-	Agg         common.Address
-	Nonce       types.BlockNonce // Nonce ensure compatibility with ethereum
-	L1Proof     L1RootHash       // the L1 block where the Parent was published
-	Root        StateRoot
-	TxHash      common.Hash // todo - include the synthetic deposits
-	Number      *big.Int    // the rollup height
-	Bloom       types.Bloom
-	ReceiptHash common.Hash
-	Extra       []byte
-	R, S        *big.Int // signature values
-	Withdrawals []Withdrawal
+	ParentHash                    L2RootHash
+	Agg                           common.Address
+	Nonce                         types.BlockNonce // Nonce ensure compatibility with ethereum
+	L1Proof                       L1RootHash       // the L1 block where the Parent was published
+	Root                          StateRoot
+	TxHash                        common.Hash // todo - include the synthetic deposits
+	Number                        *big.Int    // the rollup height
+	Bloom                         types.Bloom
+	ReceiptHash                   common.Hash
+	Extra                         []byte
+	R, S                          *big.Int // signature values
+	Withdrawals                   []Withdrawal
```

Comment:
this field will disappear once the cross chain messages are finalised, right?

Reply:
Yep, I just haven't reached the point of hot swapping the full bridge logic in order to get the simulation tests passing